internal/services/kusto/kusto_cluster_resource_test.go

```diff
@@ -400,9 +400,10 @@ resource "azurerm_kusto_cluster" "test" {
   location                      = azurerm_resource_group.test.location
   resource_group_name           = azurerm_resource_group.test.name
   public_network_access_enabled = false
+  public_ip_type = "DualStack"
```

Comment:
Maybe add a testcase to check whether this property is updateble.

Reply:
It is updatable, let me add test case later on

Reply:
test case added

`GOROOT=C:\Users\yunliu1\sdk\go1.18.1 #gosetup
GOPATH=C:\Users\yunliu1\go #gosetup
C:\Users\yunliu1\sdk\go1.18.1\bin\go.exe test -c -o D:\Temp\GoLand\___TestAccKustoCluster_update_in_github_com_hashicorp_terraform_provider_azurerm_internal_services_kusto.test.exe github.com/hashicorp/terraform-provider-azurerm/internal/services/kusto #gosetup
C:\Users\yunliu1\sdk\go1.18.1\bin\go.exe tool test2json -t D:\Temp\GoLand\___TestAccKustoCluster_update_in_github_com_hashicorp_terraform_provider_azurerm_internal_services_kusto.test.exe -test.v -test.paniconexit0 -test.run ^\QTestAccKustoCluster_update\E$ #gosetup
=== RUN   TestAccKustoCluster_update
=== PAUSE TestAccKustoCluster_update
=== CONT  TestAccKustoCluster_update
--- PASS: TestAccKustoCluster_update (3755.24s)
PASS`